Traceback (most recent call last):
  File "/Users/jvanscho/miniconda3/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/jvanscho/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/jvanscho/miniconda3/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/jvanscho/miniconda3/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/jvanscho/miniconda3/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/jvanscho/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/jvanscho/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/jvanscho/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from packaging import version
import gama
if version.parse(gama.__version__) < version.parse("20.1.0"):
    print("GAMA is outdated. Please update now!")
else:
    print("OK. You may continue :)")
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[3], line 2[0m
[1;32m      1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpackaging[39;00m [38;5;28;01mimport[39;00m version
[0;32m----> 2[0m [38;5;28;01mimport[39;00m [38;5;21;01mgama[39;00m
[1;32m      3[0m [38;5;28;01mif[39;00m version[38;5;241m.[39mparse(gama[38;5;241m.[39m__version__) [38;5;241m<[39m version[38;5;241m.[39mparse([38;5;124m"[39m[38;5;124m20.1.0[39m[38;5;124m"[39m):
[1;32m      4[0m     [38;5;28mprint[39m([38;5;124m"[39m[38;5;124mGAMA is outdated. Please update now![39m[38;5;124m"[39m)

[0;31mModuleNotFoundError[0m: No module named 'gama'
ModuleNotFoundError: No module named 'gama'

